### PR TITLE
Milan models

### PR DESCRIPTION
**Pull request by Garlyle**

### CODE REVIEW

File: app/src/main/AndroidManifest.xml [hunk 1]
```diff
@@ -5,6 +5,7 @@
   
     <uses-permission android:name="android.permission.INTERNET" />
     <uses-permission android:name="android.permission.GET_ACCOUNTS" />
+    <uses-permission android:name="android.permission.INTERNET" />
```
Comment (adamgyulavari) [inline app/src/main/AndroidManifest.xml]: @Garlyle duplicated line

File: app/src/main/java/com/greenfox/opal/gitinder/model/Profile.java [hunk 2]
```diff
@@ -0,0 +1,27 @@
+package com.greenfox.opal.gitinder.model;
+
+public class Profile {
+    String login;
+    String avatarUrl;
+    String repos;
+    String languages;
```
Comment (adamgyulavari) [inline app/src/main/java/com/greenfox/opal/gitinder/model/Profile.java]: @Garlyle `languages` field is not a String, but a String array

File: app/src/main/java/com/greenfox/opal/gitinder/model/Profile.java [hunk 3]
```diff
@@ -0,0 +1,27 @@
+package com.greenfox.opal.gitinder.model;
+
+public class Profile {
+    String login;
+    String avatarUrl;
+    String repos;
```
Comment (adamgyulavari) [inline app/src/main/java/com/greenfox/opal/gitinder/model/Profile.java]: @Garlyle `repos` field is not a String, but a String array

File: app/src/main/java/com/greenfox/opal/gitinder/model/User.java [hunk 4]
```diff
@@ -0,0 +1,22 @@
+package com.greenfox.opal.gitinder.model;
+
+public class User {
+    String username;
+    String accessToken;
+    String appToken;
```
Comment (adamgyulavari) [inline app/src/main/java/com/greenfox/opal/gitinder/model/User.java]: @Garlyle this is not called like this

**Review by adamgyulavari — CHANGES_REQUESTED** [review]

File: app/src/main/java/com/greenfox/opal/gitinder/model/Profile.java [hunk 5]
```diff
@@ -0,0 +1,29 @@
+package com.greenfox.opal.gitinder.model;
+
+import java.util.List;
+
+public class Profile {
```
Comment (adamgyulavari) [inline app/src/main/java/com/greenfox/opal/gitinder/model/Profile.java]: @Garlyle these models supposed to be responses from the server, hence these should be there

Comment (adamgyulavari) [reply app/src/main/java/com/greenfox/opal/gitinder/model/Profile.java]: also responses are models, so move the whole package under models

Comment (adamgyulavari) [reply app/src/main/java/com/greenfox/opal/gitinder/model/Profile.java]: and extend `BaseResponse`

Comment (adamgyulavari) [reply app/src/main/java/com/greenfox/opal/gitinder/model/Profile.java]: `model.response`

**Review by adamgyulavari — CHANGES_REQUESTED** [review]

File: app/src/main/java/com/greenfox/opal/gitinder/model/Profile.java (hunk 5 above)
Comment (Garlyle) [reply app/src/main/java/com/greenfox/opal/gitinder/model/Profile.java]: @adamgyulavari Done

**Review by adamgyulavari — APPROVED** [review]

**Comment by adamgyulavari** [comment]
@Garlyle resolve conflicts

**Pull request merged**